sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -135,9 +146,35 @@ struct {
 }
 ```
 
-Supported since version 3 of this specification.[^1]
+Starting in version 3 of this specification.
 </td>
-</tr><tr>
+</tr>
+<td rowspan="1">
+
+``` C++
+std::variant<ze_command_queue_handle_t,
+             ze_command_list_handle_t>
+```
+</td>
+<td>
+
+``` C++
+struct {
+  std::variant<ze_command_queue_handle_t,
+               ze_command_list_handle_t>
+    NativeHandle;
+  device Device;
+  ext::oneapi::level_zero::ownership Ownership{
+      ext::oneapi::level_zero::ownership::transfer};
+  property_list Properties{};

```

Comment:
I think it's better to be consistent across the APIs in this extension.  Since the other `make_OBJECT` APIs have an `Ownership` flag, let's keep it here too.

Reply:
In 2024 we will clean up these APIs. At that time we could consider other changes.